recipe/meta.yaml

```diff
@@ -10,11 +10,15 @@ source:
     md5: 774b74580e3cbc5b0d45c6ec345a64ae
 
 build:
-    number: 0
-    skip: true    # [win]
+    number: 1
+    features:
+        - vc9                [win and py27]
+        - vc10               [win and py34]
+        - vc14               [win and py35]
```

Comment:
So, as the linter says. These need to be like so `# [win and py27]` for example.